service/src/main/kotlin/fi/espoo/evaka/decision/DecisionAddress.kt

```diff
@@ -46,4 +48,21 @@ data class DecisionSendAddress(
     val row1: String,
     val row2: String,
     val row3: String
-)
+) {
+    companion object {
+        fun fromPerson(person: PersonData.Detailed, messageProvider: IMessageProvider, lang: String? = "fi") =
+            person.let {
+                if (addressUsable(person.streetAddress, person.postalCode, person.postOffice)) {
+                    DecisionSendAddress(
+                        person.streetAddress!!, person.postalCode!!, person.postOffice!!,
+                        person.streetAddress, "${person.postalCode} ${person.postOffice}", ""
+                    )
+                } else {
+                    when (lang) {
+                        "sv" -> messageProvider.getDefaultFeeDecisionAddress(MessageLanguage.SV)
+                        else -> messageProvider.getDefaultFeeDecisionAddress(MessageLanguage.FI)
+                    }
```

Comment:
I think this default address should be passed to this function or the whole function should be moved someplace else because `DecisionAddress` is not used only with fee decisions and the default address depends on the decision.

Reply:
Just to make it less likely that the wrong default address is used accidentally.

Reply:
Yeah, something like this https://github.com/espoon-voltti/evaka/pull/1217/commits/cdb895564ea8c47781f77699a6c7e0793eeec91d?

Reply:
Yeah that looks good 👍